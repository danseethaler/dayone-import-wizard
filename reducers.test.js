import filesReducer from './src/reducers/files';

test('adding file to initial state', () => {
  const actual = filesReducer(
    {},
    {
      type: 'ADD_FILES',
      files: [
        '/Users/danseethaler/Developer/electron/dayone-import-wizard/algorithm.txt'
      ]
    }
  );

  expect(actual).toMatchSnapshot();
});
