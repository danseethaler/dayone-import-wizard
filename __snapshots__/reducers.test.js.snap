// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`adding file to initial state 1`] = `
Object {
  "/Users/danseethaler/Developer/electron/dayone-import-wizard/algorithm.txt": Object {
    "active": true,
    "dayone": Object {
      "d": "2000-01-01T12:00",
    },
    "filePath": "/Users/danseethaler/Developer/electron/dayone-import-wizard/algorithm.txt",
    "markdown": "",
    "status": "initializing",
    "title": "algorithm",
  },
}
`;
